corehq/motech/fhir/tests/test_models.py

```diff
@@ -636,6 +640,34 @@ def test_bad_vector(self):
             self.resource_type.validate_resource(patient)
 
 
+class TestRepeaterRepeatRecordRelationship(TestCase):
+
+    def test_case_repeater_repeat_records(self):
+        repeater_id = str(uuid4())
+        fhir_repeater = SQLFHIRRepeater.objects.create(
+            domain=DOMAIN,
+            repeater_id=repeater_id,
+            fhir_version=FHIR_VERSION_4_0_1,
+        )
+        self.addCleanup(fhir_repeater.delete)
+
+        payload_id = str(uuid4())
+        repeat_record = SQLRepeatRecord.objects.create(
+            domain=DOMAIN,
+            payload_id=payload_id,
+            repeater=fhir_repeater,
+            registered_at=datetime.now(),
+        )
+        self.addCleanup(repeat_record.delete)
+
+        repeat_records = list(fhir_repeater.repeat_records.all())
+        self.assertEqual(repeat_records, [repeat_record])
+        self.assertEqual(
+            repeat_records[0].repeater.fhir_version,
```

Comment:
This line shows that you can fetch the properties of a Repeater subsubclass instance from a repeat record intuitively. You don't need to use hidden one-to-one foreign keys.